packages/openactive-integration-tests/test/templates/b-req.js

```diff
@@ -138,6 +138,23 @@ function createNoBrokerNameBReq(data) {
   return dissocPath(['broker', 'name'], req);
 }
 
+/**
+ * Paid B request with incorrect totalPaymentDue value.
+ * The price in totalPaymentDue is less than that returned in the C2 request.
+ *
+ * @param {BReqTemplateData} data
+ */
+function createIncorrectTotalPaymentDuePriceBReq(data) {
+  const req = createStandardPaidBReq(data);
+  return {
+    ...req,
+    totalPaymentDue: {
+      ...req.totalPaymentDue,
+      price: req.totalPaymentDue.price - 1,
```

Comment:
```suggestion
      price: req.totalPaymentDue.price + 1,
```

If we -1 we could end up with a negative price (as the price is in £ not pence), which is an interesting error state, but not one that we're testing for here?

Reply:
Yes! I had a dream about this, and was going to change it hah